services/navigation/builtin/builtin.go

```diff
@@ -93,16 +98,26 @@ func (conf *Config) Validate(path string) ([]string, error) {
 	}
 	deps = append(deps, conf.BaseName)
 
-	if conf.MovementSensorName == "" {
-		return nil, utils.NewConfigValidationFieldRequiredError(path, "movement_sensor")
-	}
-	deps = append(deps, conf.MovementSensorName)
-
 	if conf.MotionServiceName == "" {
 		conf.MotionServiceName = resource.DefaultServiceName
 	}
 	deps = append(deps, resource.NewName(motion.API, conf.MotionServiceName).String())
 
+	if conf.MapTypeName == "" {
```

Comment:
[note] setting default using same pattern as other parameters, however, this does not allow logging of setting default map_type